### ISSUE TITLE
Using conditional editor to add richtext to inspector overrides main editor cell

### ISSUE THREAD

**Issue by enikao** [JetBrains/MPS-extensions#268]
Example at https://github.com/enikao/ConditionalRichtextIssue

I have Language `LangA` containing `LangAChild`.

In `ExtLang`, i use conditional editors to amend the inspector of `LangAChild`.
It adds a _NodeAttribute_ `DocAttr`, which contains a `Text`.

*Expected behavior:* If I create the node attribute, the `Text` should only be rendered in the inspector (as defined in the editors).

*Actual behavior:* As soon as I add a `Text`, it replaces the rendering of `LangAChild` in the main editor.

Using MPS 2019.1.1

![image](https://user-images.githubusercontent.com/2211690/70321569-ebf95a80-1827-11ea-9a56-0366293e3e52.png)

**Comment by enikao** [comment]
Actually, it doesn't depend on using `Text` in the attribute; it happens to any kind of attribute.

**Comment by enikao** [comment]
... and I found a workaround, available in the repository in the "workaround" branch:

1. Change the editor of `DocAttr` to only contain `[> attributed node <]`.
2. Create new editor hint `insideInspector`
3. Create new editor `insideInspector for DocAttr` and add the actual editor you want
4. Change the inspector editor of the conditional editor for `LangAChild` to add this context hint

We could also use a conditional editor for `DocAttr` with the condition `editorContext.isInspector();`, but then we get annoying errors in editor aspect.

**Comment by arimer** [comment]
@alexanderpann, @slisson  do you know if there was any change trageting this issue?

**Comment by alexanderpann** [comment]
No, there was no change. The workaround is still valid but I am wondering if this is an MPS issue or an issue in our language.
